actions/workflows/st2_e2e_tests.yaml

```diff
@@ -65,14 +65,32 @@ workflows:
                     host: <% $.host %>
                     env: <% $.st2_cli_env %>
                 on-success:
-                    - robot_test
-            robot_test:
+                    - robot_docs_tests
+            robot_docs_tests:
                 action: core.remote
                 input:
                     hosts: <% $.host %>
                     env: <% $.st2_cli_env %>
-                    cmd: cd /tmp/st2tests && source venv/bin/activate && pybot robotfm_tests/
-                    timeout: 1200
+                    cmd: cd /tmp/st2tests && source venv/bin/activate && pybot robotfm_tests/docs/
+                    timeout: 300
+                on-success:
+                    - robot_mistral_tests
+            robot_mistral_tests:
+                action: core.remote
+                input:
+                    hosts: <% $.host %>
+                    env: <% $.st2_cli_env %>
+                    cmd: cd /tmp/st2tests && source venv/bin/activate && pybot robotfm_tests/mistral/
+                    timeout: 300
+                on-success:
+                    - robot_chatopsCI_tests
+            robot_chatopsCI_tests:
+                action: core.remote
+                input:
+                    hosts: <% $.host %>
+                    env: <% $.st2_cli_env %>
+                    cmd: cd /tmp/st2tests && source venv/bin/activate && pybot robotfm_tests/chatopsCI/
+                    timeout: 300
```

Comment:
Test suite took about 300 secs in all. Keeping it 300 for each feature (buffer for more test cases/suites)